modules/raft/src/test/java/org/apache/ignite/internal/raft/RaftGroupServiceTest.java

```diff
@@ -213,36 +188,29 @@ public void testRefreshLeaderElectedAfterDelay() throws Exception {
         assertEquals(NODES.get(0), service.leader());
     }
 
-    /**
-     * @throws Exception
-     */
     @Test
     public void testRefreshLeaderWithTimeout() throws Exception {
         mockLeaderRequest(true);
 
         RaftGroupService service =
-            RaftGroupServiceImpl.start(TEST_GRP, cluster, FACTORY, TIMEOUT, NODES, false, DELAY, executor).get(3, TimeUnit.SECONDS);
+                RaftGroupServiceImpl.start(TEST_GRP, cluster, FACTORY, TIMEOUT, NODES, false, DELAY, executor).get(3, TimeUnit.SECONDS);
 
         try {
             service.refreshLeader().get(500, TimeUnit.MILLISECONDS);
```

Comment:
This could be changed to `assertThrows(TimeoutException.class, ...)`

Reply:
I didn't change this code much, I only fixed the checkstyle issues. But okay, let's refactor it as well